features/Grass Lighting/Shaders/RunGrass.hlsl

```diff
@@ -2,6 +2,14 @@
 #include "Common/FrameBuffer.hlsl"
 #include "Common/MotionBlur.hlsl"
 
+struct LightingData
```

Comment:
Is this structure always around or just tied to CloudShadows?

Reply:
This is the shared buffer defined in [State.h/.cpp](https://github.com/doodlum/skyrim-community-shaders/blob/eca469a2582eb89f64184b3e45235d9fa541676c/src/State.h#L94). It has been in Lighting.hlsl but not the others. For further info, please ask @doodlum , as he used it first in the cloud shadow code of Lighting.hlsl

Reply:
![Supplementary](https://github.com/doodlum/skyrim-community-shaders/assets/76898260/cac35bdf-96ac-4e44-8545-2226d59b79ed)

Reply:
Ok. I guess you're right it should just be moved into /common since it's being setup by state.